src/scripts/fssrgen.targets

```diff
@@ -25,8 +25,8 @@ Copyright (C) Microsoft Corporation. All Rights Reserved.
     <!-- Create the output directory in case it doesn't exist yet -->
     <MakeDir Directories="$(IntermediateOutputPath)"/>
     <!-- Run the tool -->
-    <Exec Command="&quot;$([System.IO.Path]::GetFullPath('$(FsiToolPath)\$(FsiToolExe)'))&quot; --exec &quot;$(MSBuildThisFileDirectory)fssrgen.fsx&quot; &quot;%(FsSrGen.FullPath)&quot; &quot;$(IntermediateOutputPath)%(FsSrGen.Filename).fs&quot;  &quot;$(IntermediateOutputPath)%(FsSrGen.Filename).resx&quot; $(FsSrGenProjectName)" Condition="'$(OS)' != 'Unix'" />
-    <Exec Command="mono $([System.IO.Path]::GetFullPath('$(FsiToolPath)\$(FsiToolExe)')) --exec &quot;$(MSBuildThisFileDirectory)fssrgen.fsx&quot; &quot;%(FsSrGen.FullPath)&quot; &quot;$(IntermediateOutputPath)%(FsSrGen.Filename).fs&quot;  &quot;$(IntermediateOutputPath)%(FsSrGen.Filename).resx&quot; $(FsSrGenProjectName)" Condition="'$(OS)' == 'Unix'" />
+    <Exec Command="&quot;$([System.IO.Path]::GetFullPath('$(ToolsetFsiToolPath)\$(ToolsetFsiToolExe)'))&quot; --exec &quot;$(MSBuildThisFileDirectory)fssrgen.fsx&quot; &quot;%(FsSrGen.FullPath)&quot; &quot;$(IntermediateOutputPath)%(FsSrGen.Filename).fs&quot;  &quot;$(IntermediateOutputPath)%(FsSrGen.Filename).resx&quot; $(FsSrGenProjectName)" Condition="'$(OS)' != 'Unix'" />
```

Comment:
We shouldn't be executing fssrgen anymore since the SDK contains `<EmbeddedText>`, etc.